docs/static/breaking-changes-90.asciidoc

```diff
@@ -13,3 +13,34 @@ Include:
 * user impact/decription/value prop
 * link to relevant docs for more information
 
+[discrete]
+[[ssl-settings-9.0]]
+===== Changes to SSL settings in {ls} plugins
+
+We've removed deprecated SSL settings in some {ls} plugins, and have replaced them with updated settings.
+If your plugin configuration contains any of these obsolete options, the plugin may fail to start.
+
```

Comment:
```suggestion

Click the arrow beside a plugin name to see the list of settings that have been
removed and their replacements. 

```

Reply:
The expandable content is a common UI element, and we use it extensively throughout our docs.  If you think we need to call it it, here's a suggestion for how and where to do that.

Reply:
I think I'm good with skipping it. If there is plenty of prior art of us using the control, and not having any explanatory text, then I'm good